### PR TITLE
Joss dev revs r1

### LINKED ISSUE #32: Misleading exception when running a libE_MOOP script without mpirun/arguments
When I first tried to run the `libe_basic_ex.py` example, I thought that there was a bug:
```
% PYTHONPATH=. python examples/libe_basic_ex.py 
Traceback (most recent call last):
  File "…/parmoo/examples/libe_basic_ex.py", line 60, in <module>
    my_moop.solve(sim_max=30)
  File "…/parmoo/parmoo/extras/libe.py", line 715, in solve
    raise ValueError("Cannot run ParMOO + libE with less than 2 " +
ValueError: Cannot run ParMOO + libE with less than 2 workers -- aborting...
```
I then learned that the following works:
```
% PYTHONPATH=. python examples/libe_basic_ex.py --comms local --nworkers 32
```

The `ValueError` does not really explain what went wrong. I suggest to either raise a more descriptive exception (e.g. `RuntimeError: You need to specify the number of workers on the command line.`), or to default to #CPU many workers. If possible, I would personally prefer the latter.

### PR DESCRIPTION
**Pull request by thchang**
Changes to the code and documentation in response to the JOSS review of @Viech

Fixes #32

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -87,13 +87,16 @@ The easiest way to install ParMOO is via the Python package index, PyPI
 
 .. code-block:: bash
 
-    pip install [--user] parmoo
+    pip install < --user > parmoo
+
+where the braces around ``< --user >`` indicates that the ``--user`` flag is
```
Comment (wildsm) [inline README.rst:92]: Consider:
where the braces around ``< --user >`` indicate that the ``--user`` flag is
(deletes an s)

File: docs/about.rst [hunk 2]
```diff
@@ -168,6 +168,18 @@ the abstract base classes defined in ``parmoo.structs``.
...
 --------
 
+ * **Acquisition function:**
+   An *acquisition function* is our language for a family of scalarizing
+   functions, which can be used to specify targets on the Pareto front.
+   Acquisition functions may use objective scores, gradient values, and/or
+   uncertainty information in order to guide ParMOO's search for an
+   approximation to the Pareto front.
+
+    * **Ex.--** several common acquisition functions from the literature
```
Comment (wildsm) [inline docs/about.rst:178]: delete "several"?

File: docs/about.rst [hunk 3]
```diff
@@ -168,6 +168,18 @@ the abstract base classes defined in ``parmoo.structs``.
...
+      include weighted sums (averages) of objective values, the epsilon
+      constraint method, and expected hypervolume improvement, which is
+      used for multiobjective Bayesian optimization.
```
Comment (wildsm) [inline docs/about.rst:181]: cannot each of these three example scalarizations be used in BO? Is there a benefit to making the hypervolume connection to BO? If not, but a MO BO connection is really desired, could perhaps say that these are common in the BO literature?

File: docs/install.rst [hunk 4]
```diff
@@ -37,7 +37,10 @@ To install the latest release:
 
 .. code-block:: bash
 
-    pip install [--user] parmoo
+    pip install < --user > parmoo
+
+where the braces around ``< --user >`` indicates that the ``--user`` flag is
```
Comment (wildsm) [inline docs/install.rst:42]: see before, maybe "indicate"?

File: docs/quickstart.rst [hunk 5]
```diff
@@ -73,13 +73,16 @@ The easiest way to install ParMOO is via the Python package index, PyPI
 
 .. code-block:: bash
 
-    pip install [--user] parmoo
+    pip install < --user > parmoo
+
+where the braces around ``< --user >`` indicates that the ``--user`` flag is
```
Comment (wildsm) [inline docs/quickstart.rst:78]: see before, maybe "indicate"?

**Review by wildsm — COMMENTED** [review]
Add specific line comments and completed review of files. Comments are entirely grammatical.

File: README.rst (hunk 1 above)
Comment (thchang) [reply README.rst]: done

File: docs/about.rst (hunk 2 above)
Comment (thchang) [reply docs/about.rst]: done

File: docs/about.rst (hunk 3 above)
Comment (thchang) [reply docs/about.rst]: deleted BO references, it was adding nothing of value here

File: docs/index.rst [hunk 6]
```diff
@@ -16,6 +16,10 @@ ParMOO stands for "parallel multiobjective optimization".
 ParMOO can be used to solve multiobjective optimization problems (MOOPs)
 or to generate batches of simulation inputs for parallel evaluation.
 
+ParMOO is intended for scientists, engineers, practitioners, and
```
[the remark this answers is not in the document]
Comment (thchang) [reply docs/index.rst]: done, as suggested

File: docs/install.rst (hunk 4 above)
Comment (thchang) [reply docs/install.rst]: done

File: docs/quickstart.rst (hunk 5 above)
Comment (thchang) [reply docs/quickstart.rst]: done

**Review by wildsm — APPROVED** [review]

**Pull request merged**